Backend/app/rider-platform/rider-app/Main/spec/Storage/ride.yaml

```diff
@@ -50,6 +50,8 @@ Ride:
       computedPriceWithCurrency: Maybe PriceAPIEntity
       chargeableRideDistance: Maybe HighPrecMeters
       chargeableRideDistanceWithUnit: Maybe Distance
+      traveledRideDistance: Maybe HighPrecMeters
```

Comment:
You can make it `Maybe Distance`, `traveledRideDistanceWithUnit` is not required.